doc/UsersGuide/source/InputNML.inc

```diff
@@ -224,4 +230,92 @@ A sample subset of this namelist is shown below:
 Additional variables for the ``&namsfc`` namelist can be found in the ``FV3/ccpp/physics/physics/sfcsub.F``
 file.
 
+atmos_model_nml
+---------------
+
+The namelist section ``&atmos_model_nml`` contains information used by the atmosphere model.
+The variables used in ``&atmos_model_nml`` are described in :numref:`Table %s <atmos_model_nml>`.
+
+.. _atmos_model_nml:
+
+.. list-table:: *List of common variables in the *atmos_model_nml* namelist section. 
+   :widths: 10 40 15 15
+   :header-rows: 1
+
+   * - Variable Name
+     - Description
+     - Data Type
+     - Default Value
+   * - blocksize
+     - Number of columns in each ``block`` sent to the physics. OpenMP threading is done over the number of blocks.
+       For best performance this number should divide the number of grid cells per processor:
+       ``((npx-1)*(npy-1)/(layout\_x)*(layout\_y))``. A description of these variables is provided
+       `here <https://noaa-emc.github.io/FV3_Dycore_ufs-v1.1.0/html/group__Parameters__List.html>`_.
+     - integer
+     - 1
+   * - chksum_debug
+     - If true, compute checksums for all variables passed into the GFS physics, before and after each physics timestep.       This is very useful for reproducibility checking.
+     - logical
+     - .false.
+   * - dycore_only
+     - If true, only the dynamical core (and not the GFS physics) is executed when running the model,
+       essentially running the model as a solo dynamical core. 
+     - logical
+     - .false.
+   * - debug
+     - If true, turn on additional diagnostics for the atmospheric model.
+     - logical
+     - .false.
+   * - sync
+     - If true, initialize timing identifiers.
+     - logical
+     - .false.
+   * - fdiag
+     - Array with dimension ``maxhr`` = 4096 listing the diagnostic output times (in hours) for the GFS physics.
+       This can either be a list of times after initialization, or an interval if only the first entry is
+       nonzero. The default setting of 0 will result in no outputs.
+     - real
+     - 0.
+   * - fhmax
+     - The maximal forecast time for output.
+     - real
+     - 384.0
+   * - fhmaxhf
+     - The maximal forecast hour for high frequency output.
+     - real
+     - 120.0
+   * - fhout
+     - Output frequency during forecast time from 0 to ``fhmax``, or from ``fhmaxhf`` to ``fhmaxhf`` if ``fhmaxf>0``.
```

Comment:
I am not understanding this definition. Does this make sense: from fhmaxhf to fhmaxhf???
Should it be (...) or from fhmaxhf to fxmax if fhmaxhf>0?